CliWrap/Command.cs

```diff
@@ -19,7 +19,8 @@ public partial class Command(
     CommandResultValidation validation,
     PipeSource standardInputPipe,
     PipeTarget standardOutputPipe,
-    PipeTarget standardErrorPipe
+    PipeTarget standardErrorPipe,
+    bool waitForOutputProcessing
```

Comment:
Let's make this an enum `CommandExitCondition` (`.PipesClosed`, `.ProcessExited`) and call the method `WithExitCondition`

Reply:
Done. I can also add some unit tests and improve documentation, once you verify the change is ok.